src/commands/synthetics/run-tests-command.ts

```diff
@@ -185,12 +202,42 @@ export class RunTestsCommand extends Command {
     let results: Result[]
     let summary: Summary
 
+    const [_, command] = this.path ?? []
+    if (command === 'build-and-test') {
```

Comment:
Is this check really necessary?
I thought you were simply creating an alias to `run-test`.

So the user could either run `run-test` or `build-and-test` for the same command.
Simply adding `--buildCommand` as an option and do the process based on the presence of the flag.